### ISSUE TITLE
Problem while installing via pip3

### ISSUE THREAD

**Issue by Sparky-14145** [3b1b/manim#898]
When I was installing the Manim via pip3 with this command: 

```shell
pip3 install manimlib
```
I received this:

```
ERROR: Could not find a version that satisfies the requirement opencv-python==3.4.2.17 (from manimlib) (from versions: 3.4.8.29, 3.4.9.31, 4.1.2.30, 4.2.0.32)
ERROR: No matching distribution found for opencv-python==3.4.2.17 (from manimlib)
```
How to deal with it?  
And how can I make 

Something may helpful:

```shell
> uname -a
Linux sparky-14145-pc 5.5.2-1-MANJARO #1 SMP PREEMPT Tue Feb 4 19:22:35 UTC 2020 x86_64 GNU/Linux

> yay -Q opencv                                                                     ↵ 1
opencv 4.2.0-1

> pip3 show opencv-python
Name: opencv-python
Version: 4.2.0.32
Summary: Wrapper package for OpenCV python bindings.
Home-page: https://github.com/skvark/opencv-python
Author: None
Author-email: None
License: MIT
Location: /home/sparky14145/.local/lib/python3.8/site-packages
Requires: numpy
Required-by: 

> python -m manim -h
/usr/bin/python: No module named manim.__main__; 'manim' is a package and cannot be directly executed
```

**Comment by zavden** [comment]
Try `python -m pip install manimlib`

**Comment by Sparky-14145** [comment]
> Try `python -m pip install manimlib`

Failed with the same information.

**Comment by Sparky-14145** [comment]
@zavden Oops, my `pip` equals to `pip3.8`.  
Installation succeeded via `pip2.7`, but when I want to execute it...

```shell
$ python -m manim example_scenes.py SquareToCircle -pl
/usr/bin/python: No module named manim.__main__; 'manim' is a package and cannot be directly executed

$ python2.7 -m manim example_scenes.py SquareToCircle -pl
/usr/bin/python2.7: No module named manim

$ manim -h
Traceback (most recent call last):
  File "/home/sparky14145/.local/bin/manim", line 6, in <module>
    from manimlib import main
  File "/home/sparky14145/.local/lib/python2.7/site-packages/manimlib/__init__.py", line 2, in <module>
    import manimlib.config
  File "/home/sparky14145/.local/lib/python2.7/site-packages/manimlib/config.py", line 174
    print(f"Failed to render scene: {str(e)}")
                                            ^
SyntaxError: invalid syntax
```

**Comment by Aathish04** [comment]
First off:
MANIM ONLY WORKS ON PYTHON 3.7 AND ABOVE.
Python 2.7 is DEPRECATED and you should NOT be using it, especially after January 1 2020.

Now, try installing plain opencv by 
`pip3 install opencv-python` and see if that works.
Remove the line about opencv in requirements.txt and install the rest of your requirements.

**Comment by Sparky-14145** [comment]
@Aathish04 I cloned the repository to local yesterday, ,with running `python ./setup.py install --user`(my default `python` is python3.8), and that works.

**Comment by Aathish04** [comment]
> @Aathish04 I cloned the repository to local yesterday, ,with running `python ./setup.py install --user`(my default `python` is python3.8), and that works.

The repository version of manimlib is likely more often updated than the Pypi version.
Congrats!

**Comment by Mathoholic** [comment]
Hello Everbody here,
Finished with satisfying every requirement :

`pip3 install -r requirements.txt 
Ignoring pycairo: markers 'sys_platform == "win32"' don't match your environment
Ignoring pyreadline: markers 'sys_platform == "win32"' don't match your environment
Requirement already satisfied: argparse==1.4.0 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 1)) (1.4.0)
Requirement already satisfied: colour==0.1.5 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 2)) (0.1.5)
Requirement already satisfied: Pillow==5.2.0 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 3)) (5.2.0)
Requirement already satisfied: progressbar==2.5 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 4)) (2.5)
Requirement already satisfied: tqdm==4.24.0 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 5)) (4.24.0)
Requirement already satisfied: pycairo==1.17.1 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 6)) (1.17.1)
Requirement already satisfied: pydub==0.23.0 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 8)) (0.23.0)
`
NOw what I have to do? I am confused as "manim" is not suggested in terminal.

Please help.

**Comment by Aathish04** [comment]
> Hello Everbody here,
> Finished with satisfying every requirement :
> 
> `pip3 install -r requirements.txt Ignoring pycairo: markers 'sys_platform == "win32"' don't match your environment Ignoring pyreadline: markers 'sys_platform == "win32"' don't match your environment Requirement already satisfied: argparse==1.4.0 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 1)) (1.4.0) Requirement already satisfied: colour==0.1.5 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 2)) (0.1.5) Requirement already satisfied: Pillow==5.2.0 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 3)) (5.2.0) Requirement already satisfied: progressbar==2.5 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 4)) (2.5) Requirement already satisfied: tqdm==4.24.0 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 5)) (4.24.0) Requirement already satisfied: pycairo==1.17.1 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 6)) (1.17.1) Requirement already satisfied: pydub==0.23.0 in /usr/local/lib/python3.8/dist-packages (from -r requirements.txt (line 8)) (0.23.0) `
> NOw what I have to do? I am confused as "manim" is not suggested in terminal.
> 
> Please help.

First off, please make a new issue next time :)

I'm guessing you cloned the git repository, moved into the git repository and installed dependencies.

In that case, manim won't be installed as a command in the terminal.

Instead, there will be a file called `manim.py` inside the 'manim' folder.

That is your `manim` command from now on.

So, instead of `manim my_animation.py`, you will use `python3 manim.py my_animation.py`.

You can still use all the command line arguments that manim offers.

**Comment by Mathoholic** [comment]
Thanks :+1:
